app/models/membership_application.rb

```diff
@@ -133,6 +133,14 @@ def is_member?
   end
 
 
+  def editable_by?(user)
```

Comment:
This method belongs in MembershipApplicationPolicy; determining who can/cannot access something is the purpose of the Policy classes. 

This will require a bit of re-arranging of the method, and the related Spec needs to be moved into the MembershipApplicationPolicySpec as well.